src/LookingGlassConfig.ts

```diff
@@ -245,11 +267,43 @@ export class LookingGlassConfig extends EventTarget {
 	}
 
 	/**
-	 * defines the quilt resolution, only change this at start, do not change this after an XRSession has started
+	 * defines the quilt resolution, if null, it will be set based on the connected device
 	 */
 
     public get quiltResolution(): number {
```

Comment:
This assumes all our quilts are perfectly square. As we discussed on Slack, this is not true. Shouldn't this return a vector 2?

If, for whatever reason, we can't do this, we should round up to 8192x8192, 6000x6000, or 4096x4096. This is what we used to do in Unity, to keep things square, but we got feedback from a customer (I think it was Microsoft?) that this was confusing as we would either stretch the pixels or leave black pixels around the edges.